src/main/java/core/basesyntax/Application.java

```diff
@@ -2,6 +2,11 @@
 
 public class Application {
     public static void main(String[] args) {
-        // create three balls using class Lottery and print information about them in console
+        Lottery ball1 = new Lottery();
```

Comment:
Instead of creating 3 separate object please do following steps:
- Add private static final int NUMBER_OF_BALLS field in Application class which stores  number of balls which should be created (in this case 3)
- than in main method use 'for' loop which for each iteration will create Lottery object and print desired text. In this 'for' loop use NUMBER_OF_BALLS  field to limit number of created balls